sdk/aqueduct/dag.py

```diff
@@ -45,9 +45,10 @@ class DAG(BaseModel):
     # Is excluded from json serialization.
     operator_by_name: Dict[str, Operator] = {}
 
-    # The fields below must be set when publishing the workflow
+    # The fields below must be set when publishing the workflow.
```

Comment:
nit: `engine_config` no longer needs to be set, so lets just say "This field must be ..."